Dockerfile_verrazzano

```diff
@@ -24,10 +25,15 @@ RUN mkdir -p /etc/dex
 
 COPY config.dev.yaml /etc/dex/
 
-FROM $BASE_IMAGE
+# Create a directory for sqlite3 path used in config.dev.yaml
+# This is used only when the image is run using docker run
+RUN mkdir -p /var/sqlite
+
+FROM $FINAL_IMAGE
 
 COPY --from=stager --chown=1001:1001 /var/dex /var/dex
 COPY --from=stager --chown=1001:1001 /etc/dex /etc/dex
+COPY --from=stager --chown=1001:1001 /var/sqlite /var/sqlite
```

Comment:
what is the 1001:1001 user? Is that something we need to create/copy from the builder image?

Reply:
Good question, I don't know. That is the user used in the upstream Dockerfile as well. The previous final image also did not have a user 1001, and the image appears to function properly. From the user and group ids perspective, nothing has changed.